generators/chipyard/src/main/scala/ConfigFragments.scala

```diff
@@ -117,38 +109,23 @@ class WithMultiRoCCHwacha(harts: Int*) extends Config((site, here, up) => {
 })
 
 
-/**
- * Config fragment to add a small Rocket core to the system as a "control" core.
- * Used as an example of a PMU core.
- */
-class WithControlCore extends Config((site, here, up) => {
```

Comment:
Is this not possible to represent in the new system?

Reply:
This  seems to be equivalent to `WithNSmallCores(1)` in the new system

Reply:
I fixed the ControlCore config to place the SmallRocket in the last hartid

Reply:
Great thanks.